discussment-core/src/main/java/org/danekja/discussment/core/accesscontrol/service/impl/PermissionService.java

```diff
@@ -144,7 +141,7 @@ public boolean canAddPost(IDiscussionUser user, Discussion discussion) {
     public boolean canEditPost(IDiscussionUser user, Post post){
         if (user == null) {
             return false;
-        } else if (Objects.equals(user.getDiscussionUserId(), post.getUserId())) {
```

Comment:
Why this change? AFAIK these are equal with the differenc that use of Objects.equals prevents NPEs in case any of the compared objects is null.

Reply:
I don't remember my original reasoning but while `Objects.equals` is safer overall, in this case `post.getUserId()` should never be null.